contrib/styles.scss

```diff
@@ -0,0 +1,298 @@
+$base-spacing: $base-line-height * 1em;
+
+// Visual Grid
+$visual-grid: true;
+$visual-grid-color: #e6f6ff;
+$visual-grid-opacity: 0.4;
+
+// Colors
+$white: #fafafa;
+$lightblue: #d5eefb;
+$blue: #aadbf5;
+$darkblue: #41a8df;
+$darkerblue: #298ec3;
+$darkerblue: #072f44;
+
+// Base
+// ============================================
+
+body {
+  font-family: $helvetica;
+  margin: 0 0 (1.5rem * 2);
+}
+
+code {
+  display: block;
+}
+
+// Header
+// ============================================
+
+.container {
+  @include margin(null auto);
+  @include padding(null $gutter);
+  max-width: 500px;
+  position: relative;
+}
+
+.welcome-message {
+  @include clearfix();
+  @include padding($base-spacing null);
+  background-color: $blue;
+  margin-bottom: 1.5rem;
+  text-align: center;
+}
+
+.welcome-message-title {
+  font-family: "helvetica Neue", "Helvetica", sans-serif;
+  font-weight: 200;
+  line-height: $base-line-height;
+}
+
+// ============================================
+
+body {
+  color: $darkerblue;
+  line-height: $base-line-height;
+  text-align: center;
+}
+
+h3 {
+  margin: 1em;
+}
+
+pre {
+  line-height: 1.6em;
+}
+
+a,
+a:visited {
+  color: inherit;
+}
+
+.code {
+  @include outer-container();
+  background-color: $lightblue;
+  padding:0.5em;
+  text-align: center;
+}
+
+.page-section {
+  @include outer-container;
+  background-color: $lightblue;
+  line-height: $base-line-height * 1.5;
+  margin-bottom: 3em;
+  text-align: center;
+}
+
+.page-section__inner {
+  background-color: $blue;
+  font-size: 0.8em;
+  height: $base-line-height * 4rem;
+  line-height: $base-line-height;
+  margin-bottom: 2em;
+  padding: 1em;
+  text-align: left;
+}
+
+// Zero
+// ============================================
+
+.zero__alpha,
+.zero__beta {
+  @include span-columns(6);
+  height: $base-line-height * 4rem;
+  line-height: $base-line-height;
+  padding: 1em;
+  text-align: left;
+}
+
+// First
+// ============================================
+
+.first__alpha {
+  @include span-columns(1);
+}
+
+.first__beta {
+  @include span-columns(11);
+}
+
+.first__gamma {
+  @include span-columns(2);
+}
+
+.first__delta {
+  @include span-columns(10);
+}
+
+.first__epsilon {
+  @include span-columns(3);
+}
+
+.first__zeta {
+  @include span-columns(9);
+}
+
+.first__eta {
+  @include span-columns(4);
+}
+
+.first__theta {
+  @include span-columns(8);
+}
+
+.first__iota {
+  @include span-columns(5);
+}
+
+.first__kappa {
+  @include span-columns(7);
+}
+
+.first__lambda {
+  @include span-columns(6);
+}
+
+.first__mu {
+  @include span-columns(6);
+}
+
+.first__beta,
+.first__delta,
+.first__zeta,
+.first__theta,
+.first__kappa,
+.first__mu {
+  @include omega;
+}
+
+// Second
+// ============================================
+
+.second__alpha {
+  @include span-columns(4);
+}
+
+.second__beta {
+  @include span-columns(8);
+  line-height: $base-line-height * 2;
+  padding: 0;
+  text-align: center;
+}
+
+.second__beta__alpha,
```

Comment:
Selector `second__beta__alpha` should be written in lowercase with hyphens<br>Selector `second__beta__beta` should be written in lowercase with hyphens